test/suites/ekuiper/config_test.go

```diff
@@ -0,0 +1,75 @@
+package test
+
+import (
+	"edgex-snap-testing/test/utils"
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestConfigOption(t *testing.T) {
+	const (
+		defaultPort = "6379"
+		newPort     = "6000"
+		portKey     = "config.edgex.default.port"
+	)
+
+	t.Cleanup(func() {
+		utils.SnapUnset(t, ekuiperSnap, portKey)
+		utils.SnapRestart(t, ekuiperSnap)
+	})
+
+	t.Run("Test config option", func(t *testing.T) {
+		t.Log("Set and verify new EdgeX Redis Message bus port to:", newPort)
```

Comment:
The sentence is grammatically incorrect. You do not "verify new message bus port **to**" something. 
```suggestion
		t.Log("Set EdgeX Redis Message bus port to:", newPort)
```

Reply:
or
```suggestion
		t.Logf("Set EdgeX Redis Message bus port to %s and verify", newPort)
```